src/main/scad/laszlo-molnar/triagonal-agony/molnar.triagonal-agony.scad

```diff
@@ -0,0 +1,85 @@
+include <puzzlecad.scad>
+
+box_puzzle_border = 6;
+
+$burr_inset = 0.125;
+$burr_bevel = 1;
+$unit_beveled = true;
+$burr_scale = 17;
+
+pin_height = 1.0;
+pin_r = 1.6;
+pin_hole_inset = 0.2;
+
+dim = $burr_scale * 3 + box_puzzle_border * 2 + $burr_inset * 2;
+    
+*box();
+*pieces();
+*cap();
+
+module box() {
+    render(convexity = 2)
+     difference() {
+        beveled_cube([dim, dim, dim]);
```

Comment:
Random thought. What do you think about using a different, smaller bevel for the box? I tried this out when I did my print of Triagonal Agony. For example, we could change this line to

beveled_cube([dim, dim, dim], $burr_bevel = 0.5);

and make the similar change to the beveled_prism statement in triangle(). Using 0.5 would create a (visually) sharper box, and also reduces (though doesn't eliminate) the noticeable "gap" between the cap and box. [This is a purely stylistic question]

Also, note that if all the dimensions are the same, you can optionally use

beveled_cube(dim)

in place of

beveled_cube([dim, dim, dim]).